packages/core/src/root.ts

```diff
@@ -31,7 +33,7 @@ export function createRemoteRoot<
   AllowedChildrenTypes extends AllowedComponents | boolean = true
 >(
   channel: RemoteChannel,
-  {strict = true}: Options<AllowedComponents> = {},
+  {strict = true, components = []}: Options<AllowedComponents> = {},
```

Comment:
This is technically a breaking change because not providing any components used to mean "all components allowed", but now means "no components allowed". I would prefer if it continued to mean "all components allowed".

Reply:
Agree. I will update it.